website/docs/r/workspace_variable_set.html.markdown

```diff
@@ -7,7 +7,7 @@ description: |-
 
 # tfe_workspace_variable_set
 
-Adds and removes variable sets from a workspace
+Adds and removes a workspace from a variable set's scope.
```

Comment:
Changed for consistency with `tfe_project_variable_set`

Reply:
That makes sense, especially because making it consistent on the project side to read "Adds and removes variable sets from a project" could be ambiguous.